library/src/app/components/trade-confirm/trade-confirm.component.html

```diff
@@ -0,0 +1,128 @@
+<div
+  [ngClass]="{
+    'cybrid-dark-theme': (configService.config$ | async)?.theme === 'DARK'
+  }"
+>
+  <ng-container *ngIf="(isRecoverable$ | async) !== false; else error">
+    <ng-container *ngIf="quote$ | async as quote; else loading">
+      <h2 mat-dialog-title>{{ 'trade.confirm.header' | translate }}</h2>
+      <mat-dialog-content>
+        <span class="mat-hint"
+          >{{ 'trade.confirm.refresh.message' | translate }}
+          {{ ' ' + refreshInterval / 1000 + ' ' }}
```

Comment:
Does this do a floating point division or integer division?

Reply:
Integer unless the host binds something line 5000.1 ms.